### PR TITLE
Changed samples to use new MPP plugin flag

### PR DESCRIPTION
**Pull request by LepilkinaElena**

### CODE REVIEW

File: build-tools/src/main/kotlin/org/jetbrains/kotlin/benchmark/BenchmarkingPlugin.kt [hunk 1]
```diff
@@ -173,7 +173,8 @@ open class BenchmarkingPlugin: Plugin<Project> {
 
     private fun Project.configureNativeTarget(hostPreset: KotlinNativeTargetPreset) {
         kotlin.targetFromPreset(hostPreset, NATIVE_TARGET_NAME) {
-            compilations.getByName("main").kotlinOptions.freeCompilerArgs = project.compilerArgs + listOf("-l", "kotlinx-cli")
+            compilations.getByName("main").kotlinOptions.freeCompilerArgs = project.compilerArgs
+            compilations.getByName("main").enableEndorsedLibs = true
```
Comment (ilmat192) [inline build-tools/src/main/kotlin/org/jetbrains/kotlin/benchmark/BenchmarkingPlugin.kt]: It's possible to write just `compilations["main"]`

**Comment by ilmat192** [comment]
Looks like MPP plugin version needs to be updated.

**Comment by LepilkinaElena** [comment]
> Looks like MPP plugin version needs to be updated.

**Comment by ilmat192** [comment]
For samples too :)

File: build-tools/src/main/kotlin/org/jetbrains/kotlin/benchmark/BenchmarkingPlugin.kt (hunk 1 above)
Comment (LepilkinaElena) [reply build-tools/src/main/kotlin/org/jetbrains/kotlin/benchmark/BenchmarkingPlugin.kt]: It'll be rewritten when I merge review with base class for plugins.

**Pull request merged**